### PR TITLE
Move to hash URL

### PR DESCRIPTION
**Pull request by ai**
Closes https://github.com/browserslist/browsersl.ist/issues/428

Closes https://github.com/browserslist/browsersl.ist/issues/475

Also small changes:
- [x] Fix running two client’s `build` on `pnpm test`
- [x] Simplify server exports and re-use some calculations

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -15,18 +15,19 @@ const App = http.createServer(async (req, res) => {
...
 
     case '/api/browsers':
-      handleAPIBrowsers(req, res)
+      handleAPIBrowsers(req, res, url)
       break
 
     default:
-      handleStatic(req, res)
+      handleStatic(req, res, url)
       break
```
Comment (a-chabin) [inline server/index.js:31]: It seems there is an extra `url` parameter here

Comment (ai) [reply server/index.js:31]: I like to use the same API for all handlers.

File: server/handlers/api-browsers.js [hunk 2]
```diff
@@ -1,16 +1,13 @@
...
-import getBrowsers, {
-  QUERY_DEFAULTS,
-  REGION_GLOBAL
-} from '../lib/get-browsers.js'
 import { sendResponseAPI } from '../lib/send-response.js'
+import { getBrowsers } from '../lib/get-browsers.js'
 
-export default async function handleAPIBrowsers(req, res) {
+export async function handleAPIBrowsers(req, res) {
   let { searchParams: params } = new URL(req.url, `http://${req.headers.host}/`)
```
Comment (a-chabin) [inline server/handlers/api-browsers.js:7]: This duplicates `new URL` in `server/index.js`.
Let's pass the `url`, as for the `handleMain(req, res, url)`.

And we are ready for a merge :)

File: server/index.js (hunk 1 above)
Comment (a-chabin) [reply server/index.js:31]: Good! At first it seemed to me that it was not in any handler)

File: server/handlers/api-browsers.js (hunk 2 above)
Comment (ai) [reply server/handlers/api-browsers.js]: Fixed e01b5a2

**Review by a-chabin — APPROVED** [review]

**Comment by a-chabin** [comment]
not critical

https://preview-476---browserslist-vw3nqlhdtq-ue.a.run.app/?q=defaults&region=alt-sa

redirects to  😮

https://preview-476---browserslist-vw3nqlhdtq-ue.a.run.app/#q=defaults&region=alt-sa

**Comment by ai** [comment]
> it's ok?

Yes. I redirect only old `q` param. I do it to avoid moving `?utm` and other third-party params to hash.

**Pull request merged**